### DISCUSSION TITLE
RSSI values might have been fixed upstream (info)

### DISCUSSION THREAD

**Discussion by eikowagenknecht** [SukramJ/aiohomematic#944] [General]
In #178 we discussed, implemented and documented a workaround for broken RSSI values. It seems that has finally been (partially) fixed in the next Raspberrymatic release: https://github.com/jens-maus/RaspberryMatic/issues/897#issuecomment-1418258733

Jens Maus suggests to make the application of the fix dependent on the ReGa type (BYTE or INTEGER).

The original CCU3 firmware might just stay broken and I don't really see what changing the behavior would improve, so as long as no one has problems with the next Raspberrymatic firmware I'd say we shouldn't change here.

This post is mostly just to inform you that there is a change coming, so you know what might be the reason if something breaks regarding RSSIs with the next Raspberrymatic release @SukramJ.

**Comment by SukramJ** [comment]
Thanks for the info.
